fvtest/compilertriltest/IfxcmpgeReductionTest.cpp

```diff
@@ -0,0 +1,440 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2017 IBM Corp. and others
```

Comment:
Was (the contents of) this file really created in 2000?

Reply:
Good ol' copy/paste bugs. This is indeed incorrect. Fixed in eeff48c